### PR TITLE
Feature/#5 crud tricks

### PR DESCRIPTION
**Pull request by MerciMathieu**

### CODE REVIEW

File: src/Controller/HomeController.php [hunk 1]
```diff
@@ -19,4 +24,96 @@ public function index(TrickRepository $trickRepository)
...
+        $media = new Media();
+        $trick->addMedia($media);
+        $form = $this->createForm(TrickType::class, $trick);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+
+            $this->setParsedMedias($trick);
+            $trick->setAuthor($this->getUser());
+            $trick->setSlug($trick->getSlug());
```
Comment (crazymad) [inline src/Controller/HomeController.php:43]: still useful ?

File: src/Controller/HomeController.php [hunk 2]
```diff
@@ -19,4 +24,96 @@ public function index(TrickRepository $trickRepository)
...
+    /**
+     * @Route("/trick/update/{slug}", name="trick_update")
+     */
+    public function trickUpdate(Trick $trick, Request $request, EntityManagerInterface $manager)
+    {
+        $form = $this->createForm(TrickType::class, $trick);
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            $this->setParsedMedias($trick);
+            $trick->setSlug($trick->getSlug());
```
Comment (crazymad) [inline src/Controller/HomeController.php:68]: still useful ?

**Review by crazymad — CHANGES_REQUESTED** [review]

File: src/Controller/HomeController.php (hunk 1 above)
Comment (MerciMathieu) [reply src/Controller/HomeController.php:43]: yes it is actually

File: src/Controller/HomeController.php (hunk 2 above)
Comment (MerciMathieu) [reply src/Controller/HomeController.php:68]: yes it is actually

File: src/Controller/HomeController.php (hunk 1 above)
Comment (MerciMathieu) [reply src/Controller/HomeController.php]: Indeed $trick->setSlug($trick->getSlug()); wasn't necessary anymore cause it's EventListener which does it. I deleted it. 
Others lines are ok.

File: src/Controller/HomeController.php (hunk 2 above)
Comment (MerciMathieu) [reply src/Controller/HomeController.php]: Indeed $trick->setSlug($trick->getSlug()); wasn't necessary anymore cause it's EventListener which does it. I deleted it. 
Others lines are ok.

**Review by crazymad — APPROVED** [review]

**Pull request merged**